src/overlays/actors/ovl_En_Tsn/z_en_tsn.c

```diff
@@ -299,7 +299,7 @@ void func_80AE0460(EnTsn* this, PlayState* play) {
         ENTSN_SET_Z(&this->unk_1D8->actor, false);
         this->actionFunc = func_80AE0418;
     } else {
-        Actor_PickUp(&this->actor, play, GI_SEAHORSE, 2000.0f, 1000.0f);
+        Actor_PickUp(&this->actor, play, GI_SEA_HORSE_CAUGHT, 2000.0f, 1000.0f);
```

Comment:
These should all change to "seahorse" since it's used as 1 word in the text.

Reply:
See this marked as resolved but no change?

Reply:
Good catch, I 100% remember changing this to 1 word at one point, yet it seems like those changes were somehow lost to git. I'll do it again